### DISCUSSION TITLE
problem with MCDU A320NX 5.1

### DISCUSSION THREAD

**Discussion by PilotAllaham** [flybywiresim/aircraft#2770] [Help] [answered]
I updated the a320 to the last virsion 5.1  development virsion and I cant add the flight plan in INIT page and I just recive this message and nothing change .
UPLINK INSERT IN PROG 
![WhatsApp Image 2020-12-28 at 13 28 00](https://user-images.githubusercontent.com/76685947/103207969-a0bd5880-4910-11eb-8b64-fcaa8c22eae2.jpeg)

Any idea how i can fix it

**Comment by ImenesFBW** [comment, marked as answer]
Did you add your SimBrief username before requesting INIT REQUEST?

**Reply by PilotAllaham** [reply]
Yes , I’ve added the simBrief username

**Reply by Rsevenflyer** [reply]
Clear the scratch pad and type the route in as you made it in SimBreif (ex: KJFKKMIA), no separation between names.
Click the LSK1L beside CO RTE.
Click LSK2R beside INIT REQUEST.
You should be good to go.
